tests/mbt/model/ccv_model.qnt

```diff
@@ -5,16 +5,36 @@ module ccv_model {
     import Time.* from "./libraries/Time"
     import extraSpells.* from "./libraries/extraSpells"
 
-    pure val consumerChains = Set("consumer1", "consumer2", "consumer3")
+    pure val consumerChainList = List("consumer1", "consumer2", "consumer3")
+    pure val consumerChains = consumerChainList.toSet()
     pure val chains = consumerChains.union(Set(PROVIDER_CHAIN))
-    pure val unbondingPeriods = chains.mapBy(chain => 2 * Week)
+    pure val defUnbondingPeriod = 2 * Week
+    pure val unbondingPeriods = chains.mapBy(chain => defUnbondingPeriod)
+    pure val trustingPeriods = chains.mapBy(chain => defUnbondingPeriod - 1 * Hour)
     pure val ccvTimeouts = chains.mapBy(chain => 3 * Week)
 
     pure val nodes = Set("node1", "node2", "node3", "node4", "node5", "node6", "node7", "node8", "node9", "node10")
     pure val InitialValidatorSet = nodes.mapBy(node => 100)
 
-    import ccv(VscTimeout = 5 * Week, CcvTimeout = ccvTimeouts, UnbondingPeriodPerChain = unbondingPeriods, ConsumerChains = consumerChains).* from "./ccv"
+    import ccv(
+        VscTimeout = 5 * Week,
+        CcvTimeout = ccvTimeouts,
+        UnbondingPeriodPerChain = unbondingPeriods,
+        ConsumerChains = consumerChains,
+        TrustingPeriodPerChain = trustingPeriods
+    ).* from "./ccv"
+
+    type Parameters = {
+        VscTimeout: Time,
+        CcvTimeout: Chain -> Time,
+        UnbondingPeriodPerChain: Chain -> Time,
+        TrustingPeriodPerChain: Chain -> Time,
+        ConsumerChains: Set[Chain],
+        Nodes: Set[Node],
+        InitialValidatorSet: Node -> int,
+    }
 
+    var params: Parameters
```

Comment:
Not fully clear how `params` are utilized. Might be helpful to add a comment here :)

Reply:
This makes sense. Thanks for the explanation.